--- setup.py.orig	2022-10-02 20:04:08.000000000 -0700
+++ setup.py	2022-10-02 20:10:04.000000000 -0700
@@ -108,7 +108,8 @@
         self.mkpath(docdir)
         doc_builder = os.path.join(self.build_lib, 'pyshtools', 'make_docs.py')
         doc_source = '.'
-        check_call([sys.executable, doc_builder, doc_source, self.build_lib])
+        executable = os.path.join(os.environ['BUILD_PREFIX'], 'bin', 'python')
+        check_call([executable, doc_builder, doc_source, self.build_lib])
 
         print('---- ALL DONE ----')
 
